plugins/lime-plugin-locate/src/locateReducer.js

```diff
@@ -2,19 +2,19 @@ import {
 	LOCATION_LOAD_SUCCESS,
 	LOCATION_LOAD_LINKS_SUCCESS,
 	LOCATION_CHANGE_SUCCESS,
-	LOCATION_USER_SET
+	LOCATION_CHANGE,
+	LOCATION_USER_SET,
+	LOCATION_TOOGLE_EDIT,
 } from './locateConstants';
 
 
 export const initialState = {
-	station: {
-		lon: 0,
-		lat: 0
-	},
+	station: undefined,
 	user: {
 		lon: 0,
 		lat: 0
 	},
+	submitting: false,
 	isCommunity: false,
 	nodeshash: []
```

Comment:
CamelCase is used, so please correct `nodeshash` to `nodesNash`.

Reply:
renamed to nodesData, as it is not a hash